draft-ietf-quic-transport.md

```diff
@@ -196,6 +196,12 @@ Ack-eliciting Packet:
   CONNECTION_CLOSE. These cause a recipient to send an acknowledgment (see
   {{sending-acknowledgements}}).
 
+Out-of-order packet:
+
+: A packet that does not increase the largest received packet number for its
+  packet number space by exactly one. A packet can arrive out of order
+  if it is delayed or if earlier packets are lost or delayed.
+
```

Comment:
Technically, a packet can also arrive out of order if a packet number is intentionally skipped, typically for optimistic ACK attack protection.

Reply:
We're using it to elicit an ACK more quickly in the case of a single PTO packet, see #2952